app/workers/zomato_data_grabbing_worker.rb

```diff
@@ -0,0 +1,73 @@
+require 'json'
+class ZomatoDataGrabbingWorker
+  include Sidekiq::Worker
+  include Faraday
+
+  def perform(*args)
+    puts 'Start Downloading Data From Zomato...'
+
+    conn = prepare_connection
+
+    response = fetch_restaurants_list(conn)
+
+    repo_info = JSON.parse response.body
```

Comment:
Whats `repo_info`?

Anyway, the `JSON.parse` statement can definitely go into the `fetch_restaurants_list` method, and leave the main method cleaner

You also don't need the entire `repo_info` object (i guess Zomato returns something which is a repository..?), you can also extract the `repo_info['restaurants']` inside `fetch_restaurants_list`, in will provide a clearer interface where `fetch_restaurants_list` actually returns a list of restaurants...